Organizing Server Virt Manual 1: Chapter 2 is done; links missing though

cal27279 opened the pull request:
Hiya @stephamon ! Could you review and merge this when you have a chance please? It's the organization for chapter 2.
Thank you =)

cal27279 committed:
Organizing Server Virt Manual 1: Chapter 2 is done; links missing though

stephamon commented on docs/understanding-server-virtualization/understanding-hypervisors.rst:
```diff
@@ -1,14 +1,13 @@
 also refer to the physical machine on which ESXi is installed.
-__________________________________
\ No newline at end of file
+__________________________________ 
```
What are these lines at the end of each file? Did you add them or was it an automatic VScode thing?

cal27279 replied:
No, I did it.  I'll get rid of them though. Thank you for taking a look at the PR Steph. =)

stephamon reviewed (approved):
Heya, looks great! I left one question in a comment, but it shouldn't hold you back from merging this.

cal27279 committed:
Update understanding-hypervisors.rst

cal27279 committed:
Update understanding-migrations-using-vmotion.rst

cal27279 committed:
Update understanding-networking.rst

cal27279 committed:
Update understanding-snapshots.rst

cal27279 committed:
Update understanding-storage.rst

cal27279 committed:
Update understanding-vcpu.rst

cal27279 committed:
Update understanding-vmware-vcenter.rst

cal27279 committed:
Update understanding-vmware-vsphere-clusters.rst

Hiya @stephamon quick question about this symbol "®" on line 81. Should I leave it like that or is there another way to express it?

cal27279 committed:
Update what-is-a-virtual-machine.rst
